b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/decodeEditUser.js

```diff
@@ -1,24 +1,35 @@
-/*
- * Copyright (c) 2022 WSO2 LLC. (http://www.wso2.com).
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
  *
  * WSO2 LLC. licenses this file to you under the Apache License,
  * Version 2.0 (the "License"); you may not use this file except
  * in compliance with the License.
  * You may obtain a copy of the License at
  *
- *http://www.apache.org/licenses/LICENSE-2.
+ *     http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing,
  * software distributed under the License is distributed on an
  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
+ * KIND, either express or implied. See the License for the
  * specific language governing permissions and limitations
  * under the License.
  */
 
+import { setUsername } from "../../../util/util/apiUtil/decodeUser";
 import callEditUser from "../../apiCall/settings/callEditUser";
 import { commonDecode } from "../../util/apiUtil/commonDecode";
 
+/**
+ * 
+ * @param {*} session 
```

Comment:
Why do we use an asterisk here? Shall we remove it from all the other places as well if it is not needed?